src/Workspaces/CSharp/Portable/Extensions/ParenthesizedExpressionSyntaxExtensions.cs

```diff
@@ -255,6 +255,14 @@ public static bool CanRemoveParentheses(this ParenthesizedExpressionSyntax node,
                 }
             }
 
+            // (condition ? ref a : ref b ) = SomeValue, parenthesis can't be removed for when conditional expression appears at left
+            // This syntax is only allowed since C# 7.2
+            if (expression.IsKind(SyntaxKind.ConditionalExpression) &&
+                node.IsLeftSideOfAnyAssignExpression())
+            {
+                return false;
+            }
```

Comment:
do we have tests that it *does* remove properly when on the rhs?

Reply:
I have updated the PR and give one.